src/mesh/primitive.rs

```diff
@@ -139,6 +139,63 @@ pub fn create_cube_sharp(position: [f32; 3], scale: f32) -> Mesh {
     Mesh::from_triangle_faces_with_vertices_and_normals(faces, vertex_positions, vertex_normals)
 }
 
+#[allow(dead_code)]
+pub fn create_box_sharp(center: Point3<f32>, scale: [f32; 3]) -> Mesh {
```

Comment:
Wouldn't it be better if we modified `create_cube_sharp` to support the use case this is intended for?

Reply:
It's only missing the non-uniform scale.

Reply:
okay